modules/apps/asset/asset-categories-item-selector-web/src/main/java/com/liferay/asset/categories/item/selector/web/internal/display/context/SelectAssetVocabularyDisplayContext.java

```diff
@@ -103,12 +109,26 @@ public List<BreadcrumbEntry> getBreadcrumbEntries() {
 		).build();
 	}
 
-	private List<AssetVocabulary> _getAssetVocabularies() {
+	private List<AssetVocabulary> _getAssetVocabularies()
+		throws PortalException {
+
+		List<DepotEntry> depotEntries =
+			DepotEntryServiceUtil.getGroupConnectedDepotEntries(
```

Comment:
you can use `getCurrentAndGroupConnectedDepotEntries` and then you do not have to add the `groupIds.add(_themeDisplay.getScopeGroupId()); `after